### ISSUE TITLE
[BUG] "Cannot read property 'guild' of undefined" when trying to play more than one playlist

### ISSUE THREAD

**Issue by 4Lajf** [SushiBtw/discord-music-player#98]
**Bug Description**
"Cannot read property 'guild' of undefined" when trying to play more than one playlist

**Reproduction Steps**
1. Play one playlist (dosen't matter if it's from YT or Spotify)
2. Try to play another playlist

**Expected Result**
The other playlist gets added to the queue

**Attachments**
my code for the command (mostly the same as in example)
```
    if (command === 'playlist' || command === 'pl') {
            message.channel.send(`Szukam playlisty...`);
        // If maxSongs is -1, will be infinite.
        await client.player.playlist(message, {
            search: args.join(' '),
            maxSongs: -1
        });
    }
```
It's of course running in async function
```
client.on('message', async (message) => {
    const args = message.content.slice(config.prefix.length).trim().split(/ +/g);
    const command = args.shift().toLowerCase();
```

**Versioning:**
 - OS: Arch Linux
 - NPM Version 15.12.0
- Module Version 7.0.1

**Comment by SushiBtw** [comment]
I will be looking into it today.

**Comment by 4Lajf** [comment]
It happens too when there are already songs in queue and you try to add a playlist

**Comment by Bxzzy** [comment]
this also happens to me, still havent found a fix yet

**Comment by SushiBtw** [comment]
**I fixed your issue and I will be pushing the update to NPM in ~4 hours.**
I will let you know, thanks for reporting!

**Comment by SushiBtw** [comment]
Hey! Update time!
I hope everything got fixed, please check within `v7.1.0` and let me know if the error still exists!
https://www.npmjs.com/package/discord-music-player

Thanks for reporting,
Sushi!
